components/qTip/QTip.js

```diff
@@ -0,0 +1,59 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import theme from './theme.css';
+import Button, { IconButton } from '../button';
+import Overlay from '../overlay';
+import { IconArrowRightSmallOutline } from '@teamleader/ui-icons';
+
+class QTip extends PureComponent {
+  static propTypes = {
+    children: PropTypes.node.isRequired,
+    closed: PropTypes.bool,
+    highlighted: PropTypes.bool,
+    icon: PropTypes.element.isRequired,
+    onChange: PropTypes.func.isRequired,
+  };
+
+  constructor() {
+    super(...arguments);
+    this.handleToggle = ::this.handleToggle;
+  }
+
+  handleToggle() {
+    this.props.onChange();
```

Comment:
Should we trigger the onChange here? onChange is a hook by itself right? Isn't it more correct to just add the toggle as prop again instead of the onChange? Just wondering - didn't even know could do this